### ISSUE TITLE
Firefox 58 crashes with CCK2

### ISSUE THREAD

**Issue by Zeremon11** [mkaply/cck2wizard#76]
Hi,
FF 58.0.1 crashes with the CCK config. The config was created with the cck2 wizard 2.2.3.5 and **runs flawlessly on FF 57.0.4**. The Crash Reporter says only "Firefox had a problem and crashed" and gives me the options to restart or quit Firefox. This happens with **new profiles only**. (Scenario tested on Windows 7 and Windows 10.)

When I choose to restart, Firefox comes up, but **no settings from the cck.cfg are applied** nor are extensions and certificates installed. 

In one of my tests I chose to quit Firefox without restart. In the profile folder only 7 files were installed. I examined the _prefs.js_ and found that the setting "extensions.cck2.SwissPostBrand58@post.ch.firstrun" was set to "false". So I gave it a try and changed it to "true". And, oh surprise, Firefox now started without complaint, and all the CCK settings were applied, extensions and certificates installed.

Have you any idea how to solve this issue?

**Comment by mkaply** [comment]
This appears to be permissions related. I'm currently investigating.

Do you set permission for certain sites to have popups/cookies?

**Comment by Zeremon11** [comment]
Yes, we do. Should we drop these permissions or handle them by otber means?

Thanks a lot for your work with the CCK!

⁣Gesendet mit BlueMail ​

**Comment by mkaply** [comment]
You can drop them now to get by the crash. I'm still debugging.

**Comment by Zeremon** [comment]
Thanks for the tip.  Actually, when I drop the permissions everything's working as expected.
